modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -253,6 +253,11 @@
 	width: 220px;
 }
 
+.cke_panel.cke_panel,
+.cke_panel.cke_combopanel {
+	z-index: 100 !important;
```

Comment:
Hey @fortunatomaldonado, when taking a look on Clay's Dropdown, I saw they are using `1000` as z-index

![image](https://user-images.githubusercontent.com/7663513/181842008-e08fd4ae-9c9b-402b-91c7-21339006b29d.png)


maybe 100, isn't too little?

Reply:
I found `z-index: 900` to work best here. Is this what you mean?